capstone/src/main/webapp/search.js

```diff
@@ -0,0 +1,9 @@
+function autocomplete(data) {
```

Comment:
Looks like this got added into this PR after the fact. Can you send this as a separate PR?

Reply:
yep! just reverted the commit!